algebras/algebra/src/main/scala/endpoints/algebra/Tag.scala

```diff
@@ -0,0 +1,81 @@
+package endpoints.algebra
+
+import endpoints.Hashing
+
+class Tag private (
+    val name: String,
+    val description: Option[String],
+    val externalDocs: Option[ExternalDocumentationObject]
+) {
+
+  override def toString =
+    s"Tag($name, $description, $externalDocs)"
+
+  override def equals(other: Any): Boolean =
+    other match {
+      case that: Tag =>
+        name == that.name && description == that.description && externalDocs == that.externalDocs
+      case _ => false
+    }
+
+  override def hashCode(): Int =
+    Hashing.hash(name, description, externalDocs)
+
+  private[this] def copy(
+      name: String = name,
+      description: Option[String] = description,
+      externalDocs: Option[ExternalDocumentationObject] = externalDocs
+  ): Tag = new Tag(name, description, externalDocs)
+
+  def withName(name: String): Tag =
+    copy(name = name)
+
+  def withDescription(description: Option[String]): Tag =
+    copy(description = description)
+
+  def withExternalDocs(externalDocs: Option[ExternalDocumentationObject]): Tag =
+    copy(externalDocs = externalDocs)
+
+}
+
+object Tag {
+
+  def apply(
```

Comment:
Could you add a scaladoc comment explaining that this builds a tag with the given name, no description and no external documentation object?